recipes/pytest-pylint/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "pytest-pylint" %}
+{% set version = "0.8.0" %}
+{% set sha256 = "41d6f223ae5e0a0fbb0056e826ecdb8046be2a828eba55c0d4f66cbfd7d27168" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - six
+    - pylint
+    - pytest
+    - astroid
+    - isort
+    - mccabe
+    - colorama
+    - py
+    - lazy-object-proxy
+    - wrapt
+  run:
+    - python
+    - six
+    - pylint
+    - pytest
+    - astroid
+    - isort
+    - mccabe
+    - colorama
+    - py
+    - lazy-object-proxy
+    - wrapt
```

Comment:
Why are there so much more dependencies than in [`setup.py`](https://github.com/carsongee/pytest-pylint/blob/master/setup.py#L22)